src/main/webapp/course.html

```diff
@@ -269,18 +267,18 @@ <h1 class="p-4 text-center">Add Your Review</h1>
 
         <div
           id="retrieve-last-rating-container"
-          class="alert alert-light alert-dismissible fade show"
+          class="alert alert-info alert-dismissible fade show"
           role="alert"
         >
           <strong id="retrieve-last-rating-message"></strong>
-          <button
-            type="button"
-            class="close"
-            data-dismiss="alert"
-            aria-label="Close"
-          >
-            <span aria-hidden="true">&times;</span>
-          </button>
+        </div>
+
+        <div
+          id="submission-status-container"
+          class="alert alert-warning alert-dismissible fade show"
+          role="alert"
+        >
+          <strong id="submission-status-message"></strong>
```

Comment:
FYI typically we bold with css, not with strong tags

Reply:
Good to know!! Just changed that as well!